### ISSUE TITLE
Color picker menu for lights is not accessible

### ISSUE THREAD

**Issue by liamerven** [shane12300/rimworld_access#57]
## Description
The color picker menu that can be accessed from a light's widget screen does not currently work with the accessibility mod.

## Steps to Reproduce
1. Select a light source
2. Open the widget screen
3. Access the colors menu
4. Mod does not provide accessibility features for this menu

## Expected Behavior
The color picker should be navigable with keyboard and announce selections via screen reader.
